commit/plugin_e2e_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"testing"
 	"time"
 
+	ocrTypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
```

Comment:
Nit: We've been calling this `ocr3types` elsewhere, nice to stay consistent
```suggestion
	ocr3types "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
```

Reply:
there's a collision in the file :( 

We already have an import as `"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"`

Reply:
Maybe just `ocrtypes` then